### PR TITLE
app-admin/lnav: Add 0.10.1

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

File: app-admin/lnav/lnav-0.10.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+RDEPEND="
+	app-arch/bzip2:0=
+	app-arch/libarchive:=
+	>=dev-db/sqlite-3.9.0
+	dev-libs/libpcre[cxx]
+	>=net-misc/curl-7.23.0
+	sys-libs/ncurses:=[unicode(+)?]
+	sys-libs/readline:0=
+	sys-libs/zlib:0="
+DEPEND="$DEPEND $RDEPEND"
```
Comment (thesamesam) [inline app-admin/lnav/lnav-0.10.1.ebuild:31]: Why not just define RDEPEND then DEPEND? Also, ${FOO} [style](https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101). Will change for you. Thank you!

File: app-admin/lnav/lnav-0.10.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+DOCS=( AUTHORS NEWS README )
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.10.0-disable-tests.patch
+)
+
+src_prepare() {
+	default
+	eautoreconf
+	# These tests use network and expect a running sshd, so we'll just delete them.
+	echo "#! /bin/bash" > test/test_remote.sh
```
Comment (thesamesam) [inline app-admin/lnav/lnav-0.10.1.ebuild:42]: || die on external commands

**Comment by thesamesam** [comment]
Thanks!

**Comment by bowlofeggs** [comment]
Thanks for the review and notes Sam!

**Pull request closed without merging**